File: Программирование/19.05.2020/first task/2/2.cpp
/**
2. Ввести с клавиатуры значения переменных a и b. Поменять значения переменных местами, используя указатели.
*/
// Выполнил: Герасимов Д.А., П1-18

#include <iostream>
using namespace std;

int main(){
    int a, b;
        cout << "Input a: ";
	cin >> a;
	cout << "Input b: ";
	cin >> b;

	///Variable for swap
    int *pa = &a, 
	*pb = &b,
	temp;
	///Swap
	temp = *pa;
	*pa = *pb;
	*pb = temp;

	cout << "a = " << a << endl
		<< "b = " << b << endl;
    return 0;
}

File: УП/К7.1/2.py
'''
Задача №2.
Известно, что для логина часто не разрешается использовать строки содержащие пробелы.
Но пользователю нашего сервиса особенно понравилась какая-то строка.
Замените пробелы в строке на символы нижнего подчеркивания, чтобы строка
могла сгодиться для логина. Если строка состоит из одного слова, менять ничего не нужно.
'''

s = input()
s = s.replace(' ', '_')
print(s)
